client/src/client.ts

```diff
@@ -28,9 +28,14 @@ export async function activate(context: ExtensionContext) {
         );
     }
 
+    let logLevel = workspace.getConfiguration('erlang_ls').logLevel;
+    if (logLevel === "") {
```

Comment:
I think this check is not necessary since there is a non-empty `default` value for this option in the definition below.